assets/src/edit-story/components/library/panes/elements/elementsPane.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Pane } from '../shared';
+
+function ElementsPane(props) {
+  return (
+    <Pane id="library-elements-pane" {...props}>
```

Comment:
Why hard code the id here? Can't we have a helper function to generate this id as they all follow the same pattern.

Reply:
Fixed in 4be495bd. And yes, I knew I should probably generalize when I made it ;)